src/note/models.py

```diff
@@ -21,15 +21,31 @@ def get_queryset(self):
         )
 
 
-class Note (models.Model):
+class NotePriority(models.Model):
+    DEFAULT_PRIORITY_ID = 1
```

Comment:
I understand your point @lingxiaoyang, but we can change now the value of that field in the `Django` admin, modifying `NotePriority` model. I was force to make that because there are some places where is necessary to add a `Note` with the `priority` and `category` fields empty. I don't complete sure what's the best option, (fix the `ID` or fix the `NAME` value field). I think that's a team! work in order to be sure that it's the perdurable decision. Whatever the decision, I can add the changes if is necessary.